### ISSUE TITLE
Configure advanced bosh-cli v2 completion

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/orange-cf-bosh-cli#35]
See https://github.com/thomasmmitchell/bosh-complete

Steps used in POC 

```sh
curl -LO https://github.com/thomasmmitchell/bosh-complete/releases/download/v1.0.4/bosh-complete-linux
chmod +x bosh-complete-linux
eval "$(./bosh-complete-linux bash-source)"

bosh TAB

Display all 116 possibilities? (y or n)
add-blob                cr                      delete-snapshot         environments            int                     rc                      stemcells               update-runtime-config
alias-env               create-env              delete-snapshots        envs                    interpolate             recreate                stop                    upload-blobs
attach-disk             create-release          delete-stemcell         errands                 is                      releases                sync-blobs              upload-release
blobs                   cs                      delete-vm               es                      l                       remove-blob             t                       upload-stemcell
c                       ct                      delr                    event                   locks                   repack-stemcell         take-snapshot           ur
cancel-task             curl                    dels                    events                  log-in                  reset-release           task                    urc
cc                      d                       dep                     export-release          log-out                 restart                 tasks                   us
cck                     dc                      deploy                  finalize-release        login                   rs                      ts                      variables
clean-up                deld                    deployment              generate-job            logout                  run-errand              uc                      vars
cloud-check             delete-config           deployments             help                    logs                    runtime-config          ucc                     vendor-package
cloud-config            delete-deployment       diff-config             ignore                  man                     scp                     unignore                vms
cloudcheck              delete-disk             disks                   init-release            manifest                snapshots               update-cloud-config     
config                  delete-env              ds                      inspect-local-stemcell  networks                ss                      update-config           
configs                 delete-network          env                     inspect-release         orphan-disk             ssh                     update-cpi-config       
cpi-config              delete-release          environment             instances               orphaned-vms            start                   update-resurrection     

```

**Comment by ogrand** [comment]
Set in 2.1.29

**Comment by gberche-orange** [comment]
@ogrand 

Basic command completion is already included https://github.com/orange-cloudfoundry/orange-cf-bosh-cli/blob/c02b725a26196886b245136174610098064c9dc2/Dockerfile#L37

However, https://github.com/thomasmitchell/bosh-complete/commits/master show dynamic completion beyond static arguments: stemcells, instances, releases, instance groups ids, ...

**Comment by ogrand** [comment]
Closed in 1.2.30
